base_local_planner/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ catkin_package(
 add_library(base_local_planner
 	src/footprint_helper.cpp
 	src/goal_functions.cpp
+    src/heading_cost_function.cpp
```

Comment:
Made all the tabs spaces.  Most of the file was using spaces to indent instead of tabs anyways.